crates/volta-core/src/run/mod.rs

```diff
@@ -57,6 +59,17 @@ pub fn execute_tool(session: &mut Session) -> Fallible<ExitStatus> {
     command.status()
 }
 
+fn debug_tool_message<T>(tool: &str, version: Sourced<T>)
```

Comment:
A tiny suggestion that doesn't much matter, but might be a nice little win for maintainability: could we make this `version: &Sourced<T>`? I just plumbed it through and it all seems to work nicely – mostly doesn't matter where we're using it, but lets us drop one `.as_ref()` in favor of just a `&`.

Reply:
Good call :+1: